mypy/main.py

```diff
@@ -1336,12 +1336,13 @@ def set_strict_flags() -> None:
 
     # Set build flags.
     if special_opts.find_occurrences:
-        state.find_occurrences = special_opts.find_occurrences.split(".")
-        assert state.find_occurrences is not None
-        if len(state.find_occurrences) < 2:
+        _find_occurrences = tuple(special_opts.find_occurrences.split("."))
+        assert _find_occurrences is not None
```

Comment:
Surely this assertion doesn't make sense anymore? `_find_occurences` can't be `None` if it was tupled in the line immediately preceding this one, right?

```suggestion
```